KBaseReport.html

```diff
@@ -1 +1 @@
class="space"></span><span>*</span><span class="space"></span><span>* a data structure, renders the template, and saves the results to a file.</span></div><div><span class="tab"></span><span class="space"></span><span>*</span><span class="space"></span><span>* It returns the output file path in the form</span></div><div><span class="tab"></span><span class="space"></span><span>*</span><span class="space"></span><span>* { 'output_file': { 'path': '/path/to/file' } }</span></div><div><span class="tab"></span><span class="space"></span><span>*/</span></div></div><span class="tab"></span><span class="keyword">funcdef</span><span class="space"></span><span class="name" id="funcdefKBaseReport.create_report_from_template">create_report_from_template</span><span>(</span><span class="parameters"><span class="parameter"><span class="name"><a href="#typedefKBaseReport.CreateReportFromTemplateParams">CreateReportFromTemplateParams</a></span><span class="space"></span><span class="name">params</span></span></span><span>)</span><span class="space"></span><span class="keyword">returns</span><span>(</span><span class="returns"><span class="parameter"><span class="name"><a href="#typedefKBaseReport.File">File</a></span><span class="space"></span><span class="name">output_file_path</span></span></span><span>)</span><span class="space"></span><span class="keyword">authentication</span><span class="space"></span><span class="keyword">required</span><span>;</span></span></div><span>}</span><span>;</span></div><div class="index"><h2>Function Index</h2><div><span class="tab"></span><span class="name"><a href="#funcdefKBaseReport.create">create</a></span></div><div><span class="tab"></span><span class="name"><a href="#funcdefKBaseReport.create_extended_report">create_extended_report</a></span></div><div><span class="tab"></span><span class="name"><a href="#funcdefKBaseReport.create_report_from_template">create_report_from_template</a></span></div><h2>Type Index</h2><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.CreateExtendedReportParams">CreateExtendedReportParams</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.CreateParams">CreateParams</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.CreateReportFromTemplateParams">CreateReportFromTemplateParams</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.File">File</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.ReportInfo">ReportInfo</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.SimpleReport">SimpleReport</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.WorkspaceObject">WorkspaceObject</a></span></div><div><span class="tab"></span><span class="name"><a href="#typedefKBaseReport.ws_id">ws_id</a></span></div></div></body></html>
```

Comment:
Did you mean to leave this minified html file here?

Reply:
removed 👍